http/play/src/main/scala/play.scala

```diff
@@ -17,24 +17,20 @@
 package freestyle.http
 
 import freestyle._
-
-import cats.Monad
-import cats.arrow.FunctionK
 import cats.instances.future._
-
 import scala.concurrent.{ExecutionContext, Future}
 
 package play {
 
   object implicits {
 
     implicit def seqToFuture[F[_], A](prog: FreeS[F, A])(
-        implicit I: ParInterpreter[F, Future],
+        implicit I: FSHandler[F, Future],
         EC: ExecutionContext
     ): Future[A] = prog.exec[Future]
```

Comment:
Should we also add an implicit for the case where you have a `ParInterpreter` in scope?